### PR TITLE
Remove testharnessreport.js include in origin-policy tests subframe

### LINKED ISSUE #22113: Origin policy tests not running on wpt.fyi
If you visit https://wpt.fyi/results/origin-policy/content-security/valid-with-semicolon.https.html?label=master&label=experimental&aligned, you see two tests: "Prerequisite check: running on HTTPS" and "Test setup of the iframe".

If you visit https://wpt.live/origin-policy/content-security/valid-with-semicolon.https.html, you see four tests; the two additional ones are "eval must be disallowed" and "img loading must be disallowed". Those are the actual things being tested.

This leads to the unfortunate result of the dashboard being useless for origin policy: https://wpt.fyi/results/origin-policy?label=master&label=experimental&aligned shows everyone passing everything, basically.

### PR DESCRIPTION
**Pull request by stephenmcgruer**
Including the testharnessreport.js here prevents the subframe tests from being reported to the main frame, which means `wpt run` misses them. Removing it shows a bunch of previously unreported tests.

Fixes https://github.com/web-platform-tests/wpt/issues/22113

### CODE REVIEW

**Comment by stephenmcgruer** [comment]
@jgraham PTAL. I couldn't figure out a better way to do this (or to test it); open to ideas.

If `message_events` are going to be necessary for correct code, we may want to just remove the ability to set them via `setup`. I found one location using `message_events`; `intersection-observer/resources/observer-in-iframe-subframe.html`.

**Comment by stephenmcgruer** [comment]
I fired off a trigger run on chrome dev; https://community-tc.services.mozilla.com/tasks/groups/JIQr4Y0pTmSPJnTQ7ckG2Q

**Comment by stephenmcgruer** [comment]
https://wpt.fyi/results/?diff&filter=ADC&run_id=448580001&run_id=438890001 should be a comparison of the change to the closest SHA without it.

**Comment by Hexcles** [comment]
Ahh right we don't actually need `testharnessreport.js` in iframes. This looks promising.

File: tools/wptrunner/wptrunner/testharnessreport.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ window.__wptrunner_process_next_event = function() {
   var props = {output: %(output)d,
                timeout_multiplier: %(timeout_multiplier)s,
                explicit_timeout: %(explicit_timeout)s,
-               message_events: ["completion"]};
+               message_events: ["start", "test_state", "result", "completion"]};
```
Comment (Hexcles) [inline tools/wptrunner/wptrunner/testharnessreport.js]: I'd expect this change wouldn't be needed in this "new approach"?

Comment (stephenmcgruer) [reply tools/wptrunner/wptrunner/testharnessreport.js]: Damn, I thought I had removed the earlier changes in `Try different approach` :(. Well that invalidates my test run...

Comment (stephenmcgruer) [reply tools/wptrunner/wptrunner/testharnessreport.js]: Oh, should be good to review now. Ignore the second patchset touching all the OP files, I will be reverting that.

I think the WPT check results look good from a quick pass.

File: origin-policy/resources/subframe-with-origin-policy.py [hunk 2]
```diff
@@ -20,7 +20,6 @@ def main(request, response):
     <title>Origin policy subframe</title>
 
     <script src="/resources/testharness.js"></script>
-    <script src="/resources/testharnessreport.js"></script>
```
Comment (Hexcles) [inline origin-policy/resources/subframe-with-origin-policy.py]: This LGTM.

**Review by Hexcles — APPROVED** [review]

**Comment by stephenmcgruer** [comment]
@domenic - can you look at the checks for this PR (https://github.com/web-platform-tests/wpt/pull/22284/checks) and see if the diffs are what you would expect?

**Comment by domenic** [comment]
Awesome, those look exactly right!

**Pull request merged**